moc-nfs-democratic-csi/overlays/moc/zero/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: democratic-csi
+
+resources:
+- ../../../base/serviceaccounts/moc-nfs-democratic-csi-controller-sa/
+- ../../../base/serviceaccounts/moc-nfs-democratic-csi-node-sa/
+- ../../../base/deployments/moc-nfs-democratic-csi-controller/
+- ../../../base/daemonsets/moc-nfs-democratic-csi-node/
+- ../../../base/secrets/node-publish-secret-moc-nfs-csi-moc-nfs-democratic-csi/
+- ../../../base/secrets/node-stage-secret-moc-nfs-csi-moc-nfs-democratic-csi/
+- ../../../base/secrets/provisioner-secret-moc-nfs-csi-moc-nfs-democratic-csi/
+- ../../../base/secrets/controller-expand-secret-moc-nfs-csi-moc-nfs-democratic-csi/
+- ../../../base/secrets/controller-publish-secret-moc-nfs-csi-moc-nfs-democratic-csi/
```

Comment:
There should be a `kustomization.yaml` in the base that is pulling these in. And in the overlay we should point to the resources from base + additional ones added in the overlay

like here for example: https://github.com/operate-first/apps/blob/bdd8c4c6c7834c3bb154a0c810791b75ad411112/metallb/overlays/moc/zero/kustomization.yaml#L6-L7

Additionally, in any other application than `cluster-scope`, there's generally no need to keep each resource in a separate folder, they can be hosted flat, take a look here for example:
https://github.com/operate-first/apps/tree/master/metallb/base

We do that differently in `cluster-scope`, because we're kinda picky in each overlay instead of inheriting fully from the base.

Reply:
Honestly, it's mentally easier for me to keep the layout as it is right now. I've fixed the missing `kustomization.yaml` for the base directory, etc.